products/aws-lambda.md

```diff
@@ -0,0 +1,304 @@
+    link: https://aws.amazon.com/blogs/compute/announcing-c-sharp-support-for-aws-lambda/
+
+-   releaseCycle: "nodejs4.3"
+    releaseLabel: Node.js 4.3
+    releaseDate: 2016-04-07
+    support: 2020-03-05 # no Deprecation Phase 1 date
+    eol: 2020-03-05
+    link: https://aws.amazon.com/blogs/compute/node-js-4-3-2-runtime-now-available-on-lambda/
+
+-   releaseCycle: "java8"
+    releaseLabel: Java 8 (AL1)
+    releaseDate: 2015-06-15
+    support: 2023-12-31
+    eol: 2024-01-31 # estimated
+    link: https://aws.amazon.com/about-aws/whats-new/2015/06/aws-lambda-supports-java/
+
+-   releaseCycle: "python2.7"
+    releaseLabel: Python 2.7
+    releaseDate: 2015-10-08
+    support: 2021-07-15
+    eol: 2022-05-30
+    link: https://aws.amazon.com/about-aws/whats-new/2015/10/aws-lambda-supports-python-versioning-scheduled-jobs-and-5-minute-functions/
+
+-   releaseCycle: "nodejs"
+    releaseLabel: Node.js 0.10
+    releaseDate: 2014-11-13
+    support: 2016-10-31 # no Deprecation Phase 1 date
+    eol: 2016-10-31
+    link: https://aws.amazon.com/blogs/aws/run-code-cloud/
+
+---
+
+> [AWS Lambda](https://aws.amazon.com/lambda/) is an event-driven, serverless computing platform
+> provided by Amazon as a part of Amazon Web Services. It is designed to enable developers to run
+> code without provisioning or managing servers. It executes code in response to events and
+> automatically manages the computing resources required by that code.
+
+## Supported runtimes
+
+AWS Lambda supports the following runtimes:
+
+- [Node.js](/nodejs)
+- [Python](/python)
+- [Ruby](/ruby)
+- [Java](/amazon-corretto)
+- [Go](/go)
+- [.NET Core](/dotnet)
+
+## Runtime updates
+
+Runtime updates are, most of the time, backward compatible with existing functions. But, in some
+rare cases, a runtime update can negatively impact an existing function. An example of such case
+is a lamda that make use of an insecure behavior fixed by a security patch.
+
+There are three modes of runtime updates:
+
+- Auto (default): functions are automatically updated to the most recent runtime version.
+- Function update: runtime update is applied at function update.
+- Manual: runtime version is set in the function configuration.
+
+[Responsibility for applying runtime updates](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-update.html#runtime-management-shared)
+varies according to chosen runtime update mode.
+
+## Runtime deprecation
+
+Deprecation (end of support) for a runtime occurs in two phases.
+
+- Phase 1: Security patches or other updates are no longer applied, and new functions that use the
+  runtime cannot be created. Functions that use a deprecated runtime are also no longer eligible
+  for technical support.
+- Phase 2: Existing functions that use the runtime cannot be updated, and rollback to that runtime
+  is no more possible. Phase 2 starts at least 30 days after the start of Phase 1.
+
+Invocations of functions that use deprecated runtime is never blocked.
```

Comment:
The table isn't matching with the text, so how about this:

```suggestion
## Support Levels

**Standard Support**: Lambda provides security patches to the underlying runtime, along with technical support.

Deprecation (end of support) for a runtime occurs in two phases:

- Phase 1: Security patches or other updates are no longer applied, and new functions that use the
  runtime cannot be created. Functions that use a deprecated runtime are also no longer eligible
  for technical support. This phase is documented in the table above as **Deprecated Support**.
- Phase 2: Existing functions that use the runtime cannot be updated, and rollback to that runtime
  is no more possible. Phase 2 starts at least 30 days after the start of Phase 1.

Invocations of functions that use deprecated runtime is never blocked.
```

Reply:
Lgtm